packages/grid/_modules_/grid/hooks/features/virtualization/renderingState.ts

```diff
@@ -13,9 +13,9 @@ export interface InternalRenderingState {
 
 export const getInitialGridRenderingState = (): InternalRenderingState => {
   return {
-    realScroll: { left: 0, top: 0 },
+    realScroll: { left: 0, top: 0, absoluteTop: 0 },
```

Comment:
realScroll.top === realScroll.absoluteTop
as demonstrated yesterday so we can remove absoluteTop

Reply:
Removed / updated